Sources/Sentry/SentryProfiler.mm

```diff
@@ -155,7 +155,12 @@
 serializedProfileData(NSDictionary<NSString *, id> *profileData, SentryTransaction *transaction,
     SentryId *profileID, NSString *truncationReason, NSString *environment, NSString *release,
     NSDictionary<NSString *, id> *serializedMetrics, NSArray<SentryDebugMeta *> *debugMeta,
-    SentryHub *hub)
+    SentryHub *hub
+#    if SENTRY_HAS_UIKIT
+    ,
+    SentryScreenFrames *gpuData
+#    endif // SENTRY_HAS_UIKIT
+)
```

Comment:
This is ugly, I know. I have a future refactor planned to move this entire function into `SentryProfilerState`.